### DISCUSSION TITLE
Query regarding GitHub pro

### DISCUSSION THREAD

**Discussion by kannansuresh** [community/community#22735] [New to GitHub]
<p>I am planning to host blogs on GitHub. I have a personal account and organization account created.<br>
If I get pro for my personal account, will that license get reflected in the organization where I am the only member?</p>
<p>What is the general suggestion for a blogging activity? Is it better to go with an organization account or a personal account?</p>

**Comment by Jme7** [comment, marked as answer]
<p>I have no idea about your first question, but I would have blogging activity on your personal account</p>

**Comment by kannansuresh** [comment]
<p>I went ahead with your suggestions on upgrading my personal account. And I got an answer to my first question. Even if you are a pro user and you are the only member of an organization, you must pay for teams.</p>

**Comment by Jme7** [comment]
<p><a class="mention" href="/kannansuresh">@kannansuresh</a> Ok, glad to help! :slight_smile:</p>
